AuxTel/index.rst

```diff
@@ -73,8 +68,48 @@ Operations
 
    Standard-Operations/index.rst
 
+.. _table-label:
+
+.. table:: Informative AuxTel Telescope Standard Operations
+
+   +--------------------------+-------+-------------------+--------------------------+
+   |  Task                    | BLOCK | Data Verification |  Communications          |
+   +==========================+=======+===================+==========================+
+   | Daytime Checkout         | T17   | RubinTV           | #summit_announce         |
+   +--------------------------+-------+-------------------+--------------------------+
+   | Daytime Calibrations     | T18   | RubinTV           | | #summit_auxtel         |
+   |                          |       |                   | | #summit_announce       |
+   |                          |       |                   | | No dome movement       |
+   |                          |       |                   | | No AuxTel movement     |
+   +--------------------------+-------+-------------------+--------------------------+
+   | Daytime Venting          | T20   | | Chronograf      | | #summit_auxtel         |
+   |                          |       | | On-site         | | #summit_announce       |
+   |                          |       |                   | | Dome movement          |
+   |                          |       |                   | | AuxTel movement        |
+   +--------------------------+-------+-------------------+--------------------------+
+   | Preparing for on-sky     | T19   | LOVE              | | #summit_auxtel         |
+   |                          |       |                   | | #summit_announce       |
+   +--------------------------+-------+-------------------+--------------------------+
+   | Scheduler                | T19   | | RubinTV         |                          |
+   |                          |       | | LOVE            |                          |
+   |                          |       | | Cronograf       |                          |
+   +--------------------------+-------+-------------------+--------------------------+
+   | Shutdown                 |  ---  | | LOVE            | | #summit_auxtel         |
+   |                          |       | | On-site         | | #summit__daytime_task  |
+   +--------------------------+-------+-------------------+--------------------------+
+
+AuxTel operations can also be constrained by Sun altitude and time. Below an schematic view of the current default AuxTel observations workflow. 
+
+.. image:: _static/AuxTel_observation_workflow.png
+   :alt: Current AuxTel observation workflow.
+   :scale: 70
+   :align: center
+|
+
 .. _Auxiliary-Telescope-Operations-Non-standard-Operations:
 
+An observing run with AuxTel can also include a series of non-standard operations:
```

Comment:
I'm not sure this is the right description for this section. Our non-standard operations are more like a set of rare troubleshooting or know-how articles. Observing runs can include a series of engineering operations, but we're trying very hard to move away from that so I was happy to see this page was focused on the standard, scheduler-driven observations.